server/src/main/java/org/opensearch/common/settings/AbstractScopedSettings.java

```diff
@@ -496,7 +496,7 @@ public final void validate(final Settings settings, final boolean validateDepend
      * @see Setting#getSettingsDependencies(String)
      */
     public final void validate(final Settings settings, final boolean validateDependencies, final boolean validateInternalOrPrivateIndex) {
-        validate(settings, validateDependencies, false, false, validateInternalOrPrivateIndex);
+        validate(settings, validateDependencies, false, true, validateInternalOrPrivateIndex);
```

Comment:
why do we need to change this?

Reply:
We're validating whether all the settings provided here are valid & registered. This param just skips the validation for archived settings. 

https://github.com/opensearch-project/OpenSearch/blob/2013a2f5db1f197ca3a2f182948da18c19b122ce/server/src/main/java/org/opensearch/common/settings/AbstractScopedSettings.java#L534

Reply:
Line 312 in MetadataUpdateSettingsService ignore this check for archived settings for closed index. Which scenario are we covering here?

Reply:
In that case lets just call the validate method
https://github.com/opensearch-project/OpenSearch/blob/784a473b3d3891e41afca3588dd3eda7a7bd3def/server/src/main/java/org/opensearch/common/settings/AbstractScopedSettings.java#L511
 which take ignoreArchivedSettings param directly in MetadataUpdateSettingsService instead of changing here.

Reply:
Done